data/themes/darktable-elegant-grey.css

```diff
@@ -100,3 +103,8 @@
 @define-color graph_fg_active @grey_95;
 @define-color graph_grid @grey_35;
 @define-color inset_histogram alpha(@grey_95, 0.50);
+
+menu
+{
+  background-color: @grey_35;
```

Comment:
please avoid directly setting colours in themes, but use defined properties instead (`@define-color background_something @grey_whatever` then `menu { color: @background_something ;}`). It will make maintenance and derived themes creation easier.

Reply:
You're right, I will improve that.